frontend/public/style/coreos.css

```diff
@@ -290,14 +290,21 @@ input[type="radio"] {
     transform: scale(1, 1); } }
 
 .co-m-cog {
-  position: relative;
   color: #999;
   display: inline-block;
-  font-size: 14px; }
+  font-size: 14px;
+  position: relative; }
   .co-m-cog:hover {
     cursor: pointer; }
-  .co-m-cog.open .co-m-cog__icon {
+  .co-m-cog.open {
     color: #2276ad; }
+  .co-m-cog__icon {
+    color: $color-pf-black-700;
+    padding: 5px;
```

Comment:
I'd probably remove the top and bottom padding. We might want something slightly wider, too, to give a bigger click target. Maybe

```css
padding: 0 7px;
```